Travis/js/main.js

```diff
@@ -0,0 +1,154 @@
+// Create an object that has a property that is an array. Log one of the elements of that array.
+	
+	myObj={
+		myObjArray:["one","two","three"]
+	}
+
+	console.log(myObj.myObjArray[2])
+
+// Create an object that has a property that is an object. Log one of the properties of that inner object.
+
+	myObjTwo = { 
+		property1:{
+			otherProp1 : 50,
+			otherProp2	: 60
+		}
+	}
+
+	console.log(myObjTwo.property1.otherProp1)
+
+// Create an object that has a property that is a function (method). Call that method.
+
+	myObjThree = {
+		myFunct:()=>{
+			console.log("MyFunctionIsWorking");
+		}
+
+	}
+
+	myObjThree.myFunct();
+
+// Create an array that has an object in it. Log one of the properties of that object.
+
+	myArray=[
+		
+		myObjFour = {property1: 100, property2: 200}
```

Comment:
declaring a global variable inside of an array like this is an anti-pattern.  it works, so no points deducted, but you're creating random global variables that you don't need and polluting the global scope.  the idea here is just an array with an object literal:
```const myArray = [
   { property1: 100, property2: 200 }
]